src/components/TrustedCompanies/TrustedCompanies.jsx

```diff
@@ -0,0 +1,141 @@
+import {React, useState, useRef, useEffect } from 'react';
+import { Box, Typography, ImageList, ImageListItem, useMediaQuery, makeStyles } from '@material-ui/core';
+import { useTheme } from '@material-ui/core/styles';
+import ArrowForwardIosIcon from '@material-ui/icons/ArrowForwardIos';
+import ArrowBackIosIcon from '@material-ui/icons/ArrowBackIos';
+
+import Image from '../../common/Image';
+
+
+
+import twilio from '../../assets/svg/twilio.svg';
+import disney from '../../assets/svg/disney.svg';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/n4alphapesto/hirebook-frontend/pullRequest?prid=7756469)
```suggestion
import disney from "../../assets/svg/disney.svg";
```